app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ProjectsController < ApplicationController
+
+  def show
+    @project = Project.find(params[:id])
+    @challenge = @project.challenge
+    # @contestants_count = ContestantProject.where(project: @project).count
+    @contestants_count = ContestantProject.all.where("project_id = #{@project.id}").count
+  end
```

Comment:
So a couple things about whats going on in this controller: 
We want to keep our controllers slim, meaning we only want what we need in here. So the only thing that we need in here is line 4. 
Line 5 we would get rid of because in the view you would just reach through the project to get to its challenges. 
```ruby
@project.challenges.theme
```
Line 7 we do not want either because it is not our controllers job to do database logic. You would create a method in your `Project` model that would grab the count of contestants. So something like:
```ruby
def num_contestants
  contestants.count
end
```